config/clusters/2i2c-uk/lis.values.yaml

```diff
@@ -45,10 +45,21 @@ jupyterhub:
       # https://hub.docker.com/r/lisacuk/lishub-base
       name: "lisacuk/lishub-base"
       tag: "581a714"
-    memory:
-      # increase as requested via https://2i2c.freshdesk.com/a/tickets/1066
-      guarantee: 512M
-      limit: 2G
+    # Added profileList per https://2i2c.freshdesk.com/a/tickets/1066
+    profileList:
+      - display_name: "Small: ~512 MB RAM / ~0.5 CPU"
+        slug: mem_512m
+        default: true
+        kubespawner_override:
+          # increase as requested via https://2i2c.freshdesk.com/a/tickets/1066
+          mem_guarantee: 512M
+          mem_limit: 1G
+      - display_name: "Large: ~1 GB RAM / ~0.5 CPU"
+        slug: mem_1g
+        kubespawner_override:
+          # increase as requested via https://2i2c.freshdesk.com/a/tickets/1066
+          mem_guarantee: 1G
+          mem_limit: 2G
```

Comment:
I think this is a good resolution for now. They get requests == limits for memory, they still probably wont use more than one 4 CPU / 32 GB node in the cluster as they have done historically, and they are presented with something that is simple to understand.

We can optimize cloud costs from this going onwards by oversubscribing a bit, requesting less memory than the limit etc.

```suggestion
      - mem_0_4:
          display_name: 0.4 GB RAM, upto 3.485 CPUs
          kubespawner_override:
            mem_guarantee: 457785792
            mem_limit: 457785792
            cpu_guarantee: 0.054453125
            cpu_limit: 3.485
            node_selector:
              node.kubernetes.io/instance-type: n2-highmem-4
      - mem_0_9:
          display_name: 0.9 GB RAM, upto 3.485 CPUs
          kubespawner_override:
            mem_guarantee: 915571584
            mem_limit: 915571584
            cpu_guarantee: 0.10890625
            cpu_limit: 3.485
            node_selector:
              node.kubernetes.io/instance-type: n2-highmem-4
      - mem_1_7:
          display_name: 1.7 GB RAM, upto 3.485 CPUs
          kubespawner_override:
            mem_guarantee: 1831143168
            mem_limit: 1831143168
            cpu_guarantee: 0.2178125
            cpu_limit: 3.485
            node_selector:
              node.kubernetes.io/instance-type: n2-highmem-4
      - mem_3_4:
          display_name: 3.4 GB RAM, upto 3.485 CPUs
          kubespawner_override:
            mem_guarantee: 3662286336
            mem_limit: 3662286336
            cpu_guarantee: 0.435625
            cpu_limit: 3.485
            node_selector:
              node.kubernetes.io/instance-type: n2-highmem-4
```